File: reverse_arr.js

let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
let reversedArr = new Array(arr.length);

for (let i = 0; i < arr.length; i++) {
    reversedArr[i] = arr[arr.length - 1 - i];
}

console.log(reversedArr);

File: reverse_arr.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    
    int arr[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
    int n = sizeof(arr) / sizeof(arr[0])-1;
    int left=0, right=n;

    while(left < right) {
        int temp = arr[left];
        arr[left] = arr[right];
        arr[right] = temp;
        left++;
        right--;
    }

    for(int i=0; i<=n; i++) {
        printf("%d, ", arr[i]);
    }
    printf("\n");
    return 0;
}
